src/cds/src/main/kotlin/org/icpclive/cds/adapters/EmulationAdapter.kt

```diff
@@ -60,15 +59,11 @@ class EmulationAdapter(
             for (run in runs) {
                 var percentage = Random.nextDouble(0.1)
                 var timeShift = 0
-                if (run.isJudged) {
```

Comment:
why isJudged can't be val field with getter result != null?

Reply:
Hm. Just because I didn't tought it can decrease amount of code to rewrite.